### ISSUE TITLE
Cannot read property 'resourceGroups' of undefined

### ISSUE THREAD

**Issue by rodoabad** [localmed/api-mock#45]
- node 4.2.6
- api-mock 0.3.1

Just updated my api-mock to the latest one and I am now getting this when starting it.

`api-mock\lib\api-mock.js:64:29`

`walker(app, ast_json['resourceGroups']);`

**Comment by jglodek** [comment]
+1
```
info:    Enabled Cross-Origin-Resource-Sharing (CORS)
info:    	Allow-Origin: *
info:    	Allow-Methods: GET, PUT, POST, PATCH, DELETE, TRACE, OPTIONS
info:    	Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Authorization, Referer, Prefer
info:    Listening on port 3000
error:   TypeError: Cannot read property 'resourceGroups' of undefined
TypeError: Cannot read property 'resourceGroups' of undefined
    at /usr/local/lib/node_modules/api-mock/lib/api-mock.js:64:29
```

**Comment by smallcampus** [comment]
I changed the following line of the file /node_modules/api-mock/lib/api-mock.js, it works.
Seems that protagonist has been updated and the original function does not return ast format by default right now.

From: (Line 57)
```
     return protagonist.parse(data, function(error, result) {
```
To:
```
    return protagonist.parse(data, {type: "ast"}, function(error, result) {
```

**Comment by sators** [comment]
+1 Thanks for the solution @smallcampus - encountered the same issue.

info:    Enabled Cross-Origin-Resource-Sharing (CORS)
info:    	Allow-Origin: *
info:    	Allow-Methods: GET, PUT, POST, PATCH, DELETE, TRACE, OPTIONS
info:    	Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Authorization, Referer, Prefer
info:    Listening on port 3000
error:   TypeError: Cannot read property 'resourceGroups' of undefined
TypeError: Cannot read property 'resourceGroups' of undefined
    at /usr/local/lib/node_modules/api-mock/lib/api-mock.js:64:29

**Comment by mllanes** [comment]
I'm having the same issue:

error:   TypeError: Cannot read property 'resourceGroups' of undefined
TypeError: Cannot read property 'resourceGroups' of undefined

**Comment by ecordell** [comment]
Thanks for the reports and the fix! Just released a new version, `0.3.2`, with this change.

**Comment by igreulich** [comment]
I am still seeing this issue. What's more, it seemed to break once I added a second resource group. That is to say...

```
# Group Users
Resources related to users in the API.

## /users
### GET
...
# Group Organizations
Resources related to organizations in the API.

## /organizations
### GET
...
```

When my .apib file only included the Users Group api-mock worked, but once I added the Organizations Group I got this error.
